.github/workflows/ci-workflow.yml

```diff
@@ -70,3 +73,26 @@ jobs:
       with:
         name: premake-windows-${{ matrix.platform }}
         path: bin\${{ matrix.config }}\
+  source:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        config: [release]
+        platform: [x64]
+    needs: [linux, macosx, windows] # Pass all builds before creating source bundle
+    steps:
+    - name: Checkout # Need to checkout to get the package action
+      uses: actions/checkout@v2
+    - name: Download Release
+      uses: actions/download-artifact@v3
+      with:
+        name: premake-linux-${{ matrix.platform }}
+    - name: Switch Branches
+      run: git switch -C ${{ github.ref_name }}
```

Comment:
Can you explain the purpose of this switching?

Reply:
We're trying to figure out how to get this to pull down the right branch in the package script.  It was an idea that @samsinsane and I had discussed offline, but I'm honestly blanking on how to get this working.  The `git clone` command in the package script isn't playing nice.  If you've got ideas, I'd love to hear them, I'm blanking.